doc/contribute/index.rst

```diff
@@ -345,6 +345,43 @@ following exceptions:
   comment, ``//``, is not allowed.
 * Use ``/**  */`` for doxygen comments that need to appear in the documentation.
 
+The C standard reserves all identifiers that begin with an underscore
+for implementation details.
+
+.. note::
+   According to the C Standard, 7.1.3 [ISO/IEC 9899:2011]
+
+   * All identifiers that begin with an underscore and either an
+     uppercase letter or another underscore are always reserved for any
+     use.
+   * All identifiers that begin with an underscore are always reserved
+     for use as identifiers with file scope in both the ordinary and tag
+     name spaces.
+   * Each macro name in any of the following subclauses
+     (including the
+     future library directions) is reserved for use as specified if any
+     of its associated headers is included,unless explicitly stated
+     otherwise.
+   * All identifiers with external linkage (including future library
+     directions) and errno are always reserved for use as identifiers
+     with external linkage.
+
+Zephyr contains the follow name conventions to
```

Comment:
contains the following...

Reply:
I'm afraid I've not understood, you want to create a new section in our Docs with name conventions (and deprecate the one in our wiki) or update the one in the wiki ?

Reply:
deperecate wiki, create a new document for coding conventions.